crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -331,6 +331,18 @@ impl AdminInterfaceApi for RealAdminInterfaceApi {
                 self.handle_admin_request_inner(DisableApp { installed_app_id })
                     .await
             }
+            AddElements {
+                cell_id,
+                elements,
+                truncate,
+                validate,
```

Comment:
```suggestion
                truncate,
                validate,
                elements,
```

It's in this order everywhere else... : )